shared/css/conversation.css

```diff
@@ -908,8 +908,8 @@ body[platform="win"] .share-service-dropdown.overflow > .dropdown-menu-item {
 }
 
 .text-chat-disabled > .text-chat-entries {
-  /* When text chat is disabled, the entries box should be 100% height. */
-  height: 100%;
+  /* 220px is the heifht of the ads tile. */
```

Comment:
Maybe more details on the comment, like 
`/* When text chat is disabled, we want to show a tile ad with 220px height */`